src/Refinery/Tactic/Internal.hs

```diff
@@ -57,17 +58,18 @@ import Refinery.ProofState
 -- * @s@   - The state type.
 -- * @m@   - The base monad.
 -- * @a@   - The return value. This to make @'TacticT'@ a monad, and will always be @'()'@
+--
+-- One of the most important things about this type is it's 'Monad' instance. @t1 >> t2@
```

Comment:
I love this comment. Might  be worth spelling out "THIS IS DIFFERENT THAN EVERY OTHER MONAD YOU'VE EVER PLAYED WITH"